ddl/db_integration_test.go

```diff
@@ -2791,6 +2791,26 @@ func (s *testIntegrationSuite3) TestCreateTemporaryTable(c *C) {
 	tk.MustGetErrCode("create temporary table t(id int)", errno.ErrNotSupportedYet)
 
 	tk.MustExec("set @@tidb_enable_noop_functions = 1")
-	tk.MustExec("create temporary table t (id int)")
-	tk.MustQuery("show warnings").Check(testutil.RowsWithSep("|", "Warning 1105 local TEMPORARY TABLE is not supported yet, TEMPORARY will be parsed but ignored"))
+
+	// Create local temporary table.
+	tk.MustExec("set @@tidb_enable_noop_functions = 1")
```

Comment:
Why we still need to set tidb_enable_noop_functions=1

Reply:
I don't want to touch more places than necessary. For example, in this pr I just want to add the create local temporary table.
If a remove this switch, I need to update some of the tests, I prefer to do it in another PR.